metadata.json

```diff
@@ -94,6 +94,10 @@
     {
       "name": "puppet",
       "version_requirement": ">= 6.0.0 < 8.0.0"
+    },
+    {
+      "name": "unbound",
```

Comment:
I'm not sure I've ever seen a requirement different than Puppet. Is such a thing even documented? I'm not aware of any tool that reads this so I wonder what sense it makes.

Reply:
no idea, i first just anted to see if CI would break in anyway, ill drop it from here and just leave t in the readme

Reply:
+1 for removing this.

Reply:
done sorry forgot to add the change to the commit